c_glib/test/test-array.rb

```diff
@@ -141,12 +141,12 @@ def test_no_diff
 
     def test_diff
       array = build_string_array(["Start", "Shutdown", "Reboot"])
-      other_array = build_string_array(["Start", "Shutdonw", "Reboot"])
+      other_array = build_string_array(["Start", "Shutdown_", "Reboot"])
       assert_equal(<<-STRING.chomp, array.diff_unified(other_array))
 
 @@ -1, +1 @@
 -"Shutdown"
-+"Shutdonw"
++"Shutdown_"
```

Comment:
```suggestion
+"Shutdown"
```

Reply:
This didn't work. It's doing some sorting to an array. It was my initial correction and tests failed when I did it.

(I know it looks like I'm just fixing things randomly without testing stuff, but I really do try to get CI to pass before I make my initial PRs.)

Reply:
I took a look at this test again.
We should not change this because it's a test to detect difference of two arrays. The "Shutdonw" typo is for the difference.

Or we can use other entirely different word something like:

```diff
diff --git a/c_glib/test/test-array.rb b/c_glib/test/test-array.rb
index c03aecf17..4da641b20 100644
--- a/c_glib/test/test-array.rb
+++ b/c_glib/test/test-array.rb
@@ -141,12 +141,12 @@ class TestArray < Test::Unit::TestCase
 
     def test_diff
       array = build_string_array(["Start", "Shutdown", "Reboot"])
-      other_array = build_string_array(["Start", "Shutdonw", "Reboot"])
+      other_array = build_string_array(["Start", "Running", "Reboot"])
       assert_equal(<<-STRING.chomp, array.diff_unified(other_array))
 
 @@ -1, +1 @@
 -"Shutdown"
-+"Shutdonw"
++"Running"
 
       STRING
     end
```

Reply:
I'll take running.